lua/entities/gmod_wire_expression2/core/entity.lua

```diff
@@ -215,6 +215,11 @@ e2function number entity:health()
 	return this:Health()
 end
 
+e2function number entity:maxHealth()
```

Comment:
This E2 function is in the wrong file; It should be in [player.lua]() instead.
Also, it is missing the `Entity.IsPlayer` check (for example, calling it on the `NULL` entity [`noentity():maxHealth()`] would cause an exception/error about something like: attempt to call `GetMaxHealth` function a nil value).

It is obviously made wrong, so why approve it @AbigailBuccaneer ?